test/resources/demo-users/user/ns1/application-and-component.yaml

```diff
@@ -17,12 +17,13 @@ metadata:
   annotations:
     build.appstudio.openshift.io/request: configure-pac
     image.redhat.com/generate: '{"visibility": "public"}'
+    build.appstudio.openshift.io/pipeline: '{"name":"docker-build", "bundle": "latest"}'
```

Comment:
Should we have this for ns2 application-and-component as well?

Reply:
Until we don't sort out the issue with the private image in the build pipeline I don't want to change anything in ns2 (which the readme refers to).